Description: Fix flicker with help shown
 Change the refresh sequence to avoid flicker
Origin: upstream, https://github.com/Tomas-M/iotop/commit/1b36b51c72088f9e346dc5dc473653487d97cfed
Forwarded: not-needed
Last-Update: 2021-09-06

--- a/src/view_curses.c
+++ b/src/view_curses.c
@@ -841,7 +841,7 @@ donedraw:
 		move(promptx,prompty);
 	curs_set(show);
 	draw_vscroll(maxx-1,head1row?2:3,maxy-1,dispcount,saveskip);
-	refresh();
+	wnoutrefresh(stdscr);
 	if (showhelp) {
 		int rhh,rhw;
 
@@ -874,8 +874,9 @@ donedraw:
 		wresize(whelp,rhh,rhw);
 		mvwin(whelp,hy,hx);
 		view_help();
-		wrefresh(whelp);
+		wnoutrefresh(whelp);
 	}
+	doupdate();
 }
 
 static inline int curses_key(int ch) {
